ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -532,6 +537,7 @@ pub extern "C" fn resize(width: i32, height: i32) {
 pub extern "C" fn perform_updates() {
     catch_any_panic(|| {
         debug!("perform_updates");
+        prefs::free_prefs();
```

Comment:
Free the C-compatible structures we allocated to build the preferences list.

Reply:
Add a comment?